collector/pkg/component/receiver/cgoreceiver/cgoreceiver.go

```diff
@@ -63,10 +63,40 @@ func (r *CgoReceiver) Start() error {
 	// Wait for the C routine running
 	time.Sleep(2 * time.Second)
 	go r.consumeEvents()
+	r.initPageFaultEvent()
 	go r.startGetEvent()
 	return nil
 }
 
+func (r *CgoReceiver) initPageFaultEvent() {
+	var PageFaultEnabled bool = false
+	for _, value := range r.cfg.SubscribeInfo {
+		if value.Name == "tracepoint-page_fault" {
+			PageFaultEnabled = true
+			break
+		}
+	}
+
+	if !PageFaultEnabled {
+		return
+	}
+
+	pageCnt := 0
+	for {
+		var pKindlingEvent unsafe.Pointer
+		res := int(C.getPageFaultInitEvent(&pKindlingEvent))
+		if res == -1 {
+			break
+		}
+		pageCnt++
+		event := convertEvent((*CKindlingEventForGo)(pKindlingEvent))
+		r.eventChannel <- event
+		r.stats.add(event.Name, 1)
+	}
+	r.telemetry.Logger.Info("pagefault init:", zap.Int("the total number of init pagefault threads", pageCnt))
+
```

Comment:
The `zap.Int(key,value)`  is used to export the struct as a JSON object.
Please move the error msg to the first param of `logger.Info`,e.g
```go
r.telemetry.Logger.Info("pagefault init successed",zap.Int("threads_count",pageCnt))
```